group_vars/all.docker.sample

```diff
@@ -27,7 +27,7 @@ dummy:
 #ceph_osd_docker_username: ceph
 #ceph_osd_docker_imagename: daemon
 #ceph_osd_docker_extra_env: "CEPH_DAEMON=OSD_CEPH_DISK_ACTIVATE" # comma separated variables
-#ceph_osd_docker_prepare_env: "CEPH_DAEMON=OSD_CEPH_DISK_PREPARE,OSD_FORCE_ZAP=1" # comma separated variables
+#ceph_osd_docker_prepare_env: "OSD_FORCE_ZAP=1" # comma separated variables
```

Comment:
can you also edit `roles/ceph-osd/defaults/main.yml` and run `generate_group_vars_sample.sh`?